docs/content/components/header.mdx

```diff
@@ -146,3 +136,10 @@ https://designsystem.digital.gov/components/header/
 Import this Sass partial into your stylesheet
 
     @import "@nciocpl/ncids-css/scss/components/nci-header/nci-extended";
```

Comment:
Can you change the codeblock to

<pre><code>
```scss nopreview
@import "@nciocpl/ncids-css/scss/components/nci-header/nci-extended";
```
</code></pre>

to make sure that the site isn't trying to render a live preview